cli/src/commands/graph/monograph/commands/list.ts

```diff
@@ -77,16 +77,16 @@ export default (opts: BaseCommandOptions) => {
           }) satisfies OutputFile[number],
       );
       await writeFile(resolve(options.out), JSON.stringify(output));
-      process.exit(0);
+      return;
     }
 
     if (options.raw) {
-      console.log(pc.yellow('Please use the --json option. The --raw option is deprecated.'));
+      console.warn(pc.yellow('Please use the --json option. The --raw option is deprecated.'));
     }
 
     if (options.raw || options.json) {
-      console.log(filteredGraphs);
-      process.exit(0);
+      console.log(JSON.stringify(filteredGraphs));
```

Comment:
The output seemed a bit off and I couldnt parse it with jq without using JSON.stringify, also follows the same pattern of the other list functions